BUILD.md

```diff
@@ -147,6 +147,13 @@ which allows you to statically link it with GCC, if you want.
    conan export external/snappy snappy/1.1.10@
    ```
 
+Export our [Conan recipe for RocksDB](./external/rocksdb).
```

Comment:
Are we forced to use a custom recipe again? If @Bronek had a branch that updates to a newer version without a custom recipe then that might be preferable, wdyt?

Reply:
I'd rather fix 6.29.5 before updating to 8+. It's easier to backport this recipe than the set of code changes that might require.

Reply:
I agree with @thejohnfreeman - that branch with upgrade to 8.8.1 is not yet ready, and when I think it is ready it might still cause fair amount of hassle (e.g. performance tests) and uncertainty, meaning it would be some time before we can use it.